guidelines/index.html

```diff
@@ -1579,14 +1580,14 @@ <h4 id="x1-4-10-reflow"><span class="secno">達成基準 1.4.10 </span>リフロ
    </ul>
    <p>利用や意味の理解に 2 次元のレイアウトが必須である一部のコンテンツを除く。</p>
   
-   <div class="note" id="issue-container-generatedID-6"><div role="heading" class="note-title marker" id="h-note-6" aria-level="5"><span>注記</span></div><p class="">320 CSS ピクセルは、1280 CSS ピクセル幅を 400% ズームで見た場合の最初の表示幅に相当する。横スクロールになるように設計されたウェブコンテンツ (例えば、縦書きのテキスト) では、256 CSS ピクセルは、高さ 1024px を 400％ ズームで見た場合の最初の表示の高さに相当する。</p></div>
+   <div class="note" id="issue-container-generatedID-6"><div role="heading" class="note-title marker" id="h-note-6" aria-level="5"><span>注記</span></div><p class="">320 CSS ピクセルは、400% ズーム時の開始ビューポート幅である 1280 CSS ピクセル幅に相当する。横スクロールになるように設計されたウェブコンテンツ (例えば、縦書きのテキスト) の場合、256 CSS ピクセルは、400% ズーム時の開始ビューポート高さである 1024 CSS ピクセル高さに相当する。</p></div>
    
-   <div class="note" id="issue-container-generatedID-7"><div role="heading" class="note-title marker" id="h-note-7" aria-level="5"><span>注記</span></div><p class="">2 次元のレイアウトを必要とするコンテンツの例としては、画像、マップ、図解、ビデオ、ゲーム、プレゼンテーション、データテーブル、及びコンテンツを操作している間にツールバーを表示しておく必要のあるインタフェースが挙げられる。</p></div>
+   <div class="note" id="issue-container-generatedID-7"><div role="heading" class="note-title marker" id="h-note-7" aria-level="5"><span>注記</span></div><p class="">2 次元のレイアウトを必要とするコンテンツの例としては、理解のために必要な画像 (地図、図解など)、映像、ゲーム、プレゼンテーション、データテーブル (個々のセルではない)、及びコンテンツを操作している間にツールバーを表示しておく必要のあるインタフェースが挙げられる。そのようなコンテンツの部分には、2 次元スクロールを提供することが許容される。</p></div>
```

Comment:
「2 次元のレイアウトを必要とするコンテンツの例……」という文がまずあって、その後に「そのようなコンテンツの部分には、2 次元スクロールを提供することが許容される」と言っているので、ぱっと見では同じことを繰り返し言っているようにも読めます。
ただ、原文を見ると後ろの方の文は "such parts of the content" といっているので、まずある特定の1つのコンテンツ (the content) を想定した上で、その中に含まれる各部を思い浮かべているように感じます。コンテンツ全体の話ではなくコンテンツの一部に含まれるケースを言っているのかもしれません。
微妙なニュアンスの話ではあるのですが、「そのようなコンテンツの部分」を「コンテンツのそのような部分」としてはどうでしょうか。

```suggestion
   <div class="note" id="issue-container-generatedID-7"><div role="heading" class="note-title marker" id="h-note-7" aria-level="5"><span>注記</span></div><p class="">2 次元のレイアウトを必要とするコンテンツの例としては、理解のために必要な画像 (地図、図解など)、映像、ゲーム、プレゼンテーション、データテーブル (個々のセルではない)、及びコンテンツを操作している間にツールバーを表示しておく必要のあるインタフェースが挙げられる。コンテンツのそのような部分には、2 次元スクロールを提供することが許容される。</p></div>
```